shared/src/business/utilities/pdfGenerator/components/PageMetaHeaderDocket.tsx

```diff
@@ -1,30 +1,38 @@
 import React from 'react';
 
 export const PageMetaHeaderDocket = ({
+  addedDocketNumbers = [],
   docketNumber,
   // do not set this to true unless the content of the PDF also is rendering the century schoolbook font; there is a puppeteer documented here https://github.com/puppeteer/puppeteer/issues/422#issuecomment-759424240
```

Comment:
Has nothing to do with the functional changes made in this PR, but: since we're updating this file, can we add the word "issue" to the clause "there is a puppeteer [issue] documented here" in this comment?

Reply:
Sure thing! I'll merge this PR to test as-is (to avoid GitHub Actions checks re-running) but make the update in the staging PR.